smriprep/workflows/norm.py

```diff
@@ -239,7 +239,7 @@ def init_anat_norm_wf(
     norm_rpt.inputs.after_label = 'Participant'  # after
 
     ds_t1_2_tpl_report = pe.Node(
-        DerivativesDataSink(base_directory=reportlets_dir, keep_dtype=True),
+        DerivativesDataSink(base_directory=reportlets_dir, suffix='T1w'),
```

Comment:
Do you want to try the old `keep_dtype`?

Reply:
I guess I'd rather make that a separate PR, just in case it's a rabbit hole.